src/lib/utils.ts

```diff
@@ -83,3 +84,24 @@ export const walletCanEditMemo = (wallet: HDWallet): boolean => {
       return true
   }
 }
+
+export type NestedArray = PartialRecord<string, PartialRecord<string, string[]>>
+
+/**
+ * @param data - Partial, two level nested object with array of strings at leaf
+ * @param level1Id - The id into the first level of the nested object
+ * @param level2Id - The id into the second level of the nested object
```

Comment:
These are just keys, yeah?

```suggestion
 * @param level1Key - The key of the first level of the nested object
 * @param level2key - The key of the second level of the nested object
```